src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -20,8 +24,13 @@
             @ApiResponse(code = 200, message = "Successful operation")})
     public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required = true) @RequestParam(value = "driversLicense", required = true) String driversLicense, @ApiParam(value = "surcode", required = true) @RequestParam(value = "surcode", required = true) String surcode) {
         /* TODO implement the logic as in the webmethods */
+        logger.fatal("getValidOpenDFCMCase is called");
         String integer_ = "2"; // This will be changed later to accept the int value from the ORDS
         String caseDesc = "ROUTINE - PROFESSIONAL"; // This will be changed later to accept the caseDesc value from the ORDS
+
+        if (StringUtils.isEmpty(driversLicense) || StringUtils.isEmpty(surcode)) {
```

Comment:
Validation should be performed using annotation in the method header instead of checking for empty strings. You'll have to go through this with me in webMethods Designer to find out if the parameters are required, In many cases they are not.

Reply:
removed the null check for now. WIll create a task to impelement logic as in webmethod . The required params can be changed as part of the task

Reply:
Hi Nancy. I've just had a look at the webMethods for this service. There's a bunch of validation logic that has to be satisfied. 1.) The both input parameters must be present(e.g. required) and 2.) they must conform to some basic formatting requirements. I am ok with you leaving this as is for now, but you will have to work with me to implement the proper validation rules.

Reply:
sure , i ll work with you on that